parts/linux/cloud-init/artifacts/components.json

```diff
@@ -822,6 +822,31 @@
           }
         }
       }
+    },
+    {
+      "name": "containerd-wasm-shims",
+      "downloadLocation": "/usr/local/bin",
+      "downloadURIs": {
+        "default": {
+          "current": {
+            "versionsV2": [
+              {
+                "renovateTag": "<DO_NOT_UPDATE>",
+                "latestVersion": "0.3.0"
+              },
+              {
+                "renovateTag": "<DO_NOT_UPDATE>",
+                "latestVersion": "0.5.1"
+              },
+              {
+                "renovateTag": "<DO_NOT_UPDATE>",
+                "latestVersion": "0.8.0"
```

Comment:
we also want 0.15.1 now as well right?

Reply:
Yes, I am going to merge in the 0.15.1 stuff once the other PR gets merged, wanted to make sure I had everything working before I started integrating the 0.15.1 stuff